File: backend-ruby/app/models/xep_cho.rb
class XepCho < ApplicationRecord
  self.table_name = 'XepCho'
  self.primary_key = 'XepChoID'
  belongs_to :ky_thi, class_name: "KyThi", foreign_key: "KyThiID"
  belongs_to :phong_thi, class_name: "PhongThi", foreign_key: "PhongThiID"
  belongs_to :dang_ky_thi, class_name: "DangKyThi", foreign_key: "DangKyThiID"
end

File: backend-csharp/Models/XepCho.cs
namespace backend_csharp.Models;

public class XepCho
{
    public int XepChoID { get; set; }
    public int DangKyThiID { get; set; }
    public int KyThiID { get; set; }
    public int PhongThiID { get; set; }
    public string SoCho { get; set; } = string.Empty;
    public int? Hang { get; set; }
    public int? Cot { get; set; }
    public DateTime ThoiDiemXep { get; set; }
    public KyThi? KyThi { get; set; }
    public PhongThi? PhongThi { get; set; }
    public DangKyThi? DangKyThi { get; set; }
}
